src/Components/Card.js

```diff
@@ -26,6 +26,7 @@ const Card = ({
   bookmarking,
   handleBookmarking
 }) => {
+  const [isActive, setActive] = useState(false);
```

Comment:
This state is used to give a class to the Bookmark component which indicates whether the card is bookmarked or not. It is set to false by default - regardless of whether the card is bookmarked or not.